planetterp/static/css/darktheme.css

```diff
@@ -16,7 +16,13 @@ i.fas.fa-star.professor-rating:after,
 
 .navbar {
 	/* necessary to override bootstrap's .bg-primary */
-	background-color: black !important;
+	background-color: transparent !important;
+}
+
+@media(max-width: 992px) {
+	#site-navbar {
+		background-color: black !important;
+	}
```

Comment:
this `!important` isn't necessary. Each use of `!important` should be carefully considered as it can lead to difficult to debug bugs.